packages/plugin-ext/src/plugin/node/debug/plugin-node-debug-adapter-creator.ts

```diff
@@ -113,6 +113,10 @@ export class NodeDebugAdapterCreator extends PluginDebugAdapterCreator {
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         const options: any = { stdio: ['pipe', 'pipe', 2] };
 
+        if (isWindows) {
+            Object.assign(options, { windowsHide: true });
```

Comment:
Suggestion: Since `options` is of type `any` already, you should just be able to directly set the field.
```suggestion
            options.windowsHide = true;
```

Reply:
done